cloud2cloud-connector/events/subscription.go

```diff
@@ -18,7 +18,7 @@ const (
 	EventType_DevicesUnregistered EventType = "devices_unregistered"
 
 	// among all
-	EventType_SubscriptionCanceled EventType = "subscription_cancelled"
+	EventType_SubscriptionCanceled EventType = "subscription_canceled"
```

Comment:
this breaks spec  - is defined by `subscription_cancelled` OCF spec

Reply:
British vs American English, but I think I can set the locale in misspell.

Reply:
Reverted